introduction/introduction.md

```diff
@@ -0,0 +1,13 @@
+This file holds the text to accompany the [introduction](https://www.safaribooksonline.com/videos/introduction-to-innersource/9781492041504/9781492041504-video321606) video.
+This accompanying text will appear as a part of the learning path (like [this example](https://www.safaribooksonline.com/learning-paths/learning-path-lean/9781491999738/9781491946527-/part01ch01.html)).
+
+# Introduction
+
+This Learning Path gives an introduction to inner source.
```

Comment:
s/inner source/InnerSource/g

Reply:
Thanks for the suggestion.  The proposed casing is not my favorite.  I noticed it use in the Trusted Committer articles but didn't bother bringing it up.

Some background ...

... I think that we like to use inner source as both a noun and a verb.  I think that we want to spell it the same way both times.  InnerSource works great as a noun, but I really run into trouble when using it as a verb.  I don't feel good about mixed casing on verb tenses like InnerSourcing and InnerSourced, so that leaves me with calling it "inner source" everywhere.

There was a small discussion of the casing to use in [this survey](https://docs.google.com/forms/d/1Z-oe1WdER-_wzW4b0Yo5RxcRLTpf8BnJk-D_k14m65I/edit#responses).

Reply:
I know that I'm far behind the curve here and my opinion doesn't carry much (if any) weight but FWIW I'm definitely a solid -1 to InnerSource. The major reason for me is mentioned in the [doc](https://docs.google.com/document/d/1nmP4-Gj4QkkRkMFaZNrnt7KmlKVJOu0zI-wxTa-6RGo/edit) linked in the survey: parity with "open source". I also agree with @rrrutledge that when used as a verb, it's really off putting. I'd also imagine that folks evangelizing inner sourcing in any capacity would use "inner sourcing" (I know I have) and we'd therefore breed disparity right out of the gate.

IMHO, unless there's a legal reason that InnerSource needs to be used, a move to "inner source" should be pursued.

My 2c anyways :)

Reply:
I'll leave this conversation open a bit longer in case there's any more comment but at this point am thinking of resolving it without change to the text.

Reply:
Just for the record, I am also in the _inner source_ camp! I just used _InnerSource_ because we have been using is more or less consistently everywhere else, AFAIK.

Reply:
Oh, nice!  What about going back through the Trusted Committer text and changing to _inner source_ instead?